charts/cluster/prometheus_rules/cluster-ha-critical.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if not (has "CNPGClusterHACritical" .excludeRules) -}}
+alert: CNPGClusterHACritical
```

Comment:
Nit: you can define a variable with the name and use it both as name and in the condition.

Reply:
AAAAAAHhhh... to avoid repeating the name.. Got it.

Reply:
@phisco Done!